landing/app/[locale]/layout.tsx

```
@@ -0,0 +1,140 @@
+  if (!SUPPORTED_LOCALES.includes(locale)) {
+    notFound();
+  }
+
+  const faqJsonLd = buildFaqJsonLd(locale);
+
+  return (
+    <>
+      <SetLang locale={locale} />
+      <script
+        type="application/ld+json"
+        dangerouslySetInnerHTML={{ __html: JSON.stringify(orgJsonLd) }}
```

Biome (2.4.6)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)